<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillEdit</class>
 <widget class="QWidget" name="SkillEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Skills</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/icons/R2D2-Archigraphs_512x512.png</normaloff>:/icons/icons/R2D2-Archigraphs_512x512.png</iconset>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>190</y>
     <width>411</width>
     <height>426</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>description</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="description"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Die Results:</string>
       </property>
       <property name="buddy">
        <cstring>dieResults</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="dieResults"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>730</y>
     <width>341</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>274</y>
     <width>28</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter:</string>
   </property>
   <property name="buddy">
    <cstring>filterLine</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="filterLine">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>274</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="skillsTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>100</y>
     <width>408</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name:</string>
      </property>
      <property name="buddy">
       <cstring>name</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Skill Base:</string>
      </property>
      <property name="buddy">
       <cstring>base</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="name">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="base">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Rules Page:</string>
      </property>
      <property name="buddy">
       <cstring>rulesPage</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Skill Category:</string>
      </property>
      <property name="buddy">
       <cstring>category</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSpinBox" name="rulesPage">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="category"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>base</tabstop>
  <tabstop>rulesPage</tabstop>
  <tabstop>category</tabstop>
  <tabstop>description</tabstop>
  <tabstop>dieResults</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
